src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -1346,6 +1359,9 @@ default boolean isFromGuild()
      *     <br>The reaction request was attempted after the account lost {@link net.dv8tion.jda.api.Permission#MESSAGE_MANAGE Permission.MESSAGE_MANAGE}
      *         in the {@link net.dv8tion.jda.api.entities.TextChannel TextChannel} when removing the reaction.</li>
      *
+     *     <li>{@link net.dv8tion.jda.api.requests.ErrorResponse#UNKNOWN_EMOJI UNKNOWN_EMOJI}
+     *     <br>The provided emote was deleted or doesn't exist.</li>
```

Comment:
I think this code can also mean that the emote is not available in the channel:

https://github.com/discord/discord-api-docs/issues/1350